pybleau/app/model/tests/test_multi_df_dataframe_analyzer.py

```diff
@@ -140,6 +140,24 @@ def test_concat_to_source_df_misaligned(self):
                              np.nan, 5])
         assert_array_equal(analyzer.source_df["NEW_COL2"].values, expected)
 
+    def test_concat_to_source_df_overlapping_col(self):
+        analyzer = self.analyzer_klass(_source_dfs={"a": self.df,
+                                                    "b": self.df3})
+        initial_a_values = self.df["a"].values
+        new_df = pd.DataFrame({"NEW_COL": range(0, 22, 2),
+                               "a": range(11)}, index=self.df.index)
```

Comment:
May want to give this "a" column a different set of values from the original "a" column, so that you can ensure the original "a" isn't just being replaced with the same values.

Reply:
Great point! Done.